stubs/setuptools/setuptools/command/test.pyi

```diff
@@ -1,43 +1,16 @@
-from _typeshed import Incomplete, Unused
-from collections.abc import Callable
-from types import ModuleType
-from typing import ClassVar, Generic, TypeVar, overload
-from typing_extensions import Self
-from unittest import TestLoader, TestSuite
+from typing import ClassVar
+from typing_extensions import deprecated
 
 from .. import Command
 
-_T = TypeVar("_T")
-_R = TypeVar("_R")
-
-class ScanningLoader(TestLoader):
-    def __init__(self) -> None: ...
-    def loadTestsFromModule(self, module: ModuleType, pattern: Incomplete | None = None) -> list[TestSuite]: ...  # type: ignore[override]
-
-class NonDataProperty(Generic[_T, _R]):
-    fget: Callable[[_T], _R]
-    def __init__(self, fget: Callable[[_T], _R]) -> None: ...
-    @overload
-    def __get__(self, obj: None, objtype: Unused = None) -> Self: ...
-    @overload
-    def __get__(self, obj: _T, objtype: Unused = None) -> _R: ...
-
-class test(Command):
-    description: str
+@deprecated(
+    "The test command is disabled and references to it are deprecated. Please remove any references to `setuptools.command.test` in all supported versions of the affected package."
```

Comment:
I would also personally prefer a multiline string with line continuation markers (`\`) to avoid going over the 130 character limit.
```suggestion
    """The test command is disabled and references to it are deprecated. \
Please remove any references to `setuptools.command.test` in all supported versions of the affected package."""
```
or
```suggestion
    """\
The test command is disabled and references to it are deprecated. \
Please remove any references to `setuptools.command.test` in all supported versions of the affected package.\
"""
```